strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/pypi/PypiBrowsePackageHtmlResponseBuilder.java

```diff
@@ -0,0 +1,69 @@
+package org.carlspring.strongbox.controllers.layout.pypi;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiArtifactCoordinates;
+import org.carlspring.strongbox.providers.io.RepositoryFiles;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.repository.Repository;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * Helper class for {@link PypiArtifactController}
+ * 
+ * @author ankit.tomar
+ */
+@Component
+public class PypiBrowsePackageHtmlResponseBuilder
+{
+
+    private final Logger logger = LoggerFactory.getLogger(PypiBrowsePackageHtmlResponseBuilder.class);
+
+    public String getHtmlResponse(List<Path> filePaths,
+                                  String packageName,
+                                  Repository repository)
+    {
+        String htmlResponse = "<html>\n"
+                + "        <head>\n"
+                + "            <title>Links for " + packageName + "</title>\n"
+                + "        </head>\n"
+                + "        <body>\n"
+                + "            <h1>Links for " + packageName + "</h1>\n"
+                + "                   " + getPackageLinks(filePaths, repository)
+                + "        </body>\n"
+                + "    </html>";
```

Comment:
Cosmetic: Could you please move all the `+` signs to be at the end of the line and re-indent the multi-line String to look like this:
```java
String foo = "foo" +
             "blah" + 
             "more blah";
````

Could you please apply this change across the rest of the pull request as well? Thanks! :)

Reply:
Fixed